packages/core/src/asciidoctor-core-api.js

```diff
@@ -3047,3 +3047,381 @@ Opal.Asciidoctor.SyntaxHighlighterBase = SyntaxHighlighterBase
+Table.prototype.getRowcount = function () {
+  return this.getAttribute('rowcount')
+}
+
+/**
+ * Get the number of columns in the table
+ * @returns {string|undefined}
+ */
+Table.prototype.getColcount = function () {
+  return this.getAttribute('colcount')
+}
+
+// Table Column
+
+/**
+ * Methods to manage the columns of an AsciiDoc table. In particular, it
+ * keeps track of the column specs
+ * @namespace
+ * @extends AbstractNode
+ */
+var Column = Opal.Asciidoctor.Table.Column
+
+/**
+ * Create a new Column element
+ * @param {Table} table
+ * @param {number} index
+ * @param {Object|undefined} attributes
+ * @returns Column
+ */
+Column.create = function (table, index, attributes) {
+  return this.$new(table, index, toHash(attributes))
+}
+
+/**
+ * Get the attribute with the given name. If the attribute is not
+ * set, undefined will be returned
+ * @param {string} name the name of the attribute
+ * @returns {string|undefined}
+ */
+Column.prototype.getAttribute = function (name) {
+  var value = this.getAttributes()[name]
+  return value === Opal.nil ? undefined : value
+}
+
+/**
+ * Get the colnumber of this cell
+ * @returns {string|undefined}
+ */
+Column.prototype.getColnumber = function () {
+  return this.getAttribute('colnumber')
+}
+
+/**
+ * Get the width of this cell
+ * @returns {string|undefined}
+ */
+Column.prototype.getWidth = function () {
+  return this.getAttribute('width')
+}
+
+/**
+ * Get the hAlign of this cell
+ * @returns {string|undefined}
+ */
+Column.prototype.getHAlign = function () {
+  return this.getAttribute('halign')
+}
+
+/**
+ * Get the vAlign of this cell
+ * @returns {string|undefined}
+ */
+Column.prototype.getVAlign = function () {
+  return this.getAttribute('valign')
+}
+
+/**
+ * Get the style of this cell
+ * @returns {string}
+ */
+Column.prototype.getStyle = function () {
+  var style = this.style
+  return style === Opal.nil ? undefined : style
+}
+
+// Table Cell
+
+/**
+ * Methods for managing the a cell in an AsciiDoc table.
+ * @namespace
+ * @extends AbstractBlock
+ */
+var Cell = Opal.Asciidoctor.Table.Cell
+
+/**
+ * Create a new Cell element
+ * @param {Column} column
+ * @param {string} cellText
+ * @param {Object|undefined} attributes
+ * @param {Object|undefined} opts
+ * @returns {Cell}
+ */
+Cell.create = function (column, cellText, attributes, opts) {
+  return this.$new(column, cellText, toHash(attributes), toHash(opts))
+}
+
+/**
+ * Get the attribute with the given name. If the attribute is not
+ * set, undefined will be returned
+ * @param {string} name the name of the attribute
+ * @returns {string|undefined}
+ */
+Cell.prototype.getAttribute = function (name) {
+  var value = this.getAttributes()[name]
+  return value === Opal.nil ? undefined : value
+}
+
+/**
+ * Get the colspan of this {@link Cell} node
+ * @memberof Cell
+ * @returns {number}  An Integer of the number of columns this cell will span (default undefines)
+ */
+Cell.prototype.getColspan = function () {
```

Comment:
If we apply the same rules, then `getColumnSpan`

Reply:
Changed to getColumnSpan